src/test/java/org/junitpioneer/jupiter/ReportEntryExtensionTests.java

```diff
@@ -572,32 +572,33 @@ void repeated_disabled() {
 		@ValueSource(strings = { "Open here I flung the shutter, when, with many a flirt and flutter,",
 				"In there stepped a stately Raven of the saintly days of yore;" })
 		@ReportEntry("{0}")
-		void parameterized_basic(String line) {
+		void parameterized_basic(@SuppressWarnings("unused") String line) {
 		}
 
 		@ParameterizedTest
 		@ValueSource(strings = { "Not the least obeisance made he; not a minute stopped or stayed he;" })
 		@ReportEntry("{0}, {1}")
-		void parameterized_unresolved(String line) {
+		void parameterized_unresolved(@SuppressWarnings("unused") String line) {
 		}
 
 		@ParameterizedTest
 		@ValueSource(strings = { "But, with mien of lord or lady, perched above my chamber door—" })
 		@ReportEntry(key = "{0}", value = "Perched upon a bust of Pallas just above my chamber door—")
-		void parameterized_key_fail(String line) {
+		void parameterized_key_fail(@SuppressWarnings("unused") String line) {
```

Comment:
I think for test classes it's ok to suppress the warning on the class level to cut down on these annotations everywhere.